styles/program.css

```
@@ -0,0 +1,302 @@
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  height: 85px;
+  min-height: 85px;
+  box-sizing: border-box;
+  padding: 0 20px;
+  text-align: right;
+  font-family: monospace;
+  background-color: rgba(172, 171, 177, 0.11);
+}
+
+.header-container {
+  -webkit-box-align: center;
+  align-items: center;
+  box-shadow: none;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  -webkit-box-flex: 0;
+  height: 100%;
+  -webkit-box-pack: start;
+  justify-content: flex-start;
+  max-height: none;
+  max-width: 1280px;
+  width: 100%;
+  z-index: 0;
+  position: static;
+  background: none 0% 0% / auto transparent;
+  flex: 0 1 100%;
+  margin: auto;
+  padding: 0 px;
+  background-color: #dedbe72c;
+}
+
+.logo-container {
+  display: flex;
+  -webkit-box-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  align-items: center;
+  height: 60%;
+  border: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+  font-size: 100%;
+  font-style: normal;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  vertical-align: baseline;
+}
+
+.header-nav {
+  display: flex;
+  -webkit-box-pack: end;
+  justify-content: flex-end;
+  -webkit-box-align: center;
+  align-items: center;
+  -webkit-box-flex: 1;
+  flex-grow: 1;
+  flex-direction: row;
+}
+
+.nav-link1 {
+  display: inline-flex;
+  -webkit-box-align: center;
+  align-items: center;
+  height: 100%;
+  color: rgb(127, 127, 127);
+  font-size: 12px;
+  line-height: 160%;
+  text-transform: uppercase;
+  padding: 10px 20px;
+  font-family: "Work Sans", sans-serif;
+}
+
+.nav-link2 {
+  display: inline-flex;
+  -webkit-box-align: center;
+  align-items: center;
+  height: 100%;
+  color: rgb(127, 127, 127);
+  font-size: 12px;
+  line-height: 160%;
+  text-transform: uppercase;
+  padding: 10px 20px;
+  font-family: "work sans", sans-serif;
+}
+
+.nav-link3 {
+  display: inline-flex;
+  -webkit-box-align: center;
+  align-items: center;
+  height: 100%;
+  color: rgb(127, 127, 127);
+  font-size: 12px;
+  line-height: 160%;
+  text-transform: uppercase;
+  padding: 10px 20px;
+  font-family: "Work Sans", sans-serif;
+}
+
+.nav-link4 {
+  display: inline-flex;
+  -webkit-box-align: center;
+  align-items: center;
+  height: 100%;
+  color: rgb(127, 127, 127);
+  font-size: 12px;
+  line-height: 160%;
+  text-transform: uppercase;
+  padding: 10px 20px;
+  font-family: "Work Sans", sans-serif;
+}
+
+.nav-link5 {
+  display: inline-flex;
+  -webkit-box-align: center;
+  align-items: center;
+  height: 100%;
+  color: rgb(127, 127, 127);
+  font-size: 10px;
+  line-height: 160%;
+  text-transform: uppercase;
+  padding: 10px 20px;
+  font-family: "Work Sans", sans-serif;
+}
+
+.nav-link6 {
+  display: inline-flex;
+  -webkit-box-align: center;
+  align-items: center;
+  height: 100%;
+  color: rgb(127, 127, 127);
+  font-size: 10px;
+  line-height: 100%;
+  text-transform: uppercase;
+  padding: 10px 20px;
+  font-family: "Work Sans", sans-serif;
+}
```

As I mentioned before in the .html file, this code can be drastically reduced by maintaining just a single style and reusing this for every link. This reduces your code by almost 75 (repeated) lines.
```suggestion
.nav-link {
  display: inline-flex;
  -webkit-box-align: center;
  align-items: center;
  height: 100%;
  color: rgb(127, 127, 127);
  font-size: 12px;
  line-height: 160%;
  text-transform: uppercase;
  padding: 10px 20px;
  font-family: "Work Sans", sans-serif;
}
```
Another way to tackle this can be by reusing the .css code but with different classes. You can achieve this by doing the following:
```suggestion
.nav-link1, .nav-link2, .nav-link3, .nav-link4, .nav-link5, .nav-link6  {
  display: inline-flex;
  -webkit-box-align: center;
  align-items: center;
  height: 100%;
  color: rgb(127, 127, 127);
  font-size: 12px;
  line-height: 160%;
  text-transform: uppercase;
  padding: 10px 20px;
  font-family: "Work Sans", sans-serif;
}
```
However, I would suggest using the first example. Because there isn't any benefit for using a different class for every link 😄